src/components/AddBareMetalHosts/AddBareMetalHosts.tsx

```diff
@@ -29,23 +27,27 @@ const AddBareMetalHosts: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
   };
 
   return (
-    <>
-      <ClusterBreadcrumbs clusterName={cluster.name} />
-      <PageSection variant={PageSectionVariants.light} isMain>
+    <Card>
+      <CardTitle>
+        <Title headingLevel="h2" size="lg" className="card-title">
+          Bare Metal Inventory
+        </Title>
+      </CardTitle>
+      <CardBody>
         <BaremetalInventoryAddHosts cluster={cluster} />
-      </PageSection>
-      <AlertsSection />
-      <ClusterToolbar>
-        <ToolbarButton
-          variant={ButtonVariant.primary}
-          name="install"
-          onClick={handleHostsInstall}
-          isDisabled={!isKnownHost(cluster)}
-        >
-          Install ready hosts
-        </ToolbarButton>
-      </ClusterToolbar>
-    </>
+        <AlertsSectionGroup />
+        <Toolbar id="cluster-toolbar">
+          <ToolbarButton
+            variant={ButtonVariant.primary}
+            name="install"
+            onClick={handleHostsInstall}
+            isDisabled={!isKnownHost(cluster)}
+          >
+            Install ready hosts
```

Comment:
This might be out of scope of this PR, but WDYT of at least the UI reflecting the `Known` status of hosts as either `Ready` or `Ready to install` instead? I left [a comment here](https://docs.google.com/document/d/1jxNMTlotmJ0GFZ1GUEQ3hfOOVLzMT6mvY8Ufo5RdErY/edit?disco=AAAAGlR2YNc) about this a while back and still feel like it would be clearer to the user that the hosts are ready/green to proceed. `Known` sounds like a backend state.

![image](https://user-images.githubusercontent.com/9122899/98062119-5d1c0600-1e1b-11eb-8111-d9936b3958fb.png)

Reply:
We can change it on UI in a follow-up. Consistently for both Day 1 and 2.

Reply:
https://github.com/mareklibra/facet-lib/pull/288

Reply:
Nice, thank you! 👍